sdk/telemetry/test/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.0
 
 require (
 	github.com/stretchr/testify v1.9.0
-	go.opentelemetry.io/auto/sdk v0.0.0-00010101000000-000000000000
+	go.opentelemetry.io/auto/sdk v0.16.0-alpha
```

Comment:
Just want to make sure I understood this correctly:
We have the `replace` for local builds/tests and this entry here will be used once the API is using this module?

Reply:
Correct. The replace only applies to this test module when using it directly. If this were ever imported the importer would not have this replace statement and would use `go.opentelemetry.io/auto/sdk v0.16.0-alpha` as an indirect dependency.